Sources/Templates/Extensions/TypeName+Extension.swift

```diff
@@ -96,3 +99,32 @@ private extension TypeName {
         return ".\(firstCase.name)(\(associatedValue))"
     }
 }
+
+// OpaqueType helpers
+extension TypeName {
+
+    var isOpaqueType: Bool {
+        name
+            .split(separator: " ")
+            .map { String($0) }
+            .contains { Constants.opaqueKeywords.contains($0) }
+    }
+
+    func wrapOptionalIfNeeded() -> String {
+        isOpaqueType && isOptional ? "(\(unwrappedTypeName))?" : name
+    }
+
+    func dropOpaqueKeywordIfNeeded() -> String {
```

Comment:
Same about naming above. This verb feels like you're mutating the string itself instead of returning.